### DISCUSSION TITLE
Show DuckDB version in a file

### DISCUSSION THREAD

**Discussion by szarnyasg** [duckdb/duckdb-wasm#1494] [Ideas]
The repository should contain the corresponding DuckDB version in a file.

Currently, it can be retrieved via using the submodule if one looks up the Git tags but this is not ideal.

**Comment by ixxie** [comment]
It would also be really great if this was visible in the README; currently, the tag in the README shows the latest version of DuckDB regardless of the one used:

![image](https://github.com/duckdb/duckdb-wasm/assets/20320695/1c3b6717-e184-448f-87ab-0b48bb45c50c)

```html
  <a href="https://github.com/duckdb/duckdb/releases/">
    <img src="https://img.shields.io/github/v/release/duckdb/duckdb?color=brightgreen&display_name=tag&logo=duckdb&logoColor=white" alt="Latest DuckDB Release">
  </a>
```

**Comment by ixxie** [comment]
Additionally, you could also consider exporting a variable called `DATABASE_VERSION` similar to `PACKAGE_VERSION` available for DuckDB WASM itself. Identical version parts for `_MAJOR`, `_MINOR` and `_PATCH` would be handy too.
